internal/zinc-apiinfo/src/main/scala/sbt/internal/inc/ClassToAPI.scala

```diff
@@ -553,8 +561,10 @@ object ClassToAPI {
   private[this] def exceptionTypes(c: Constructor[_]): Array[Type] = c.getGenericExceptionTypes
 
   private[this] def exceptionTypes(m: Method): Array[Type] = m.getGenericExceptionTypes
-  private[this] def parameterTypes(m: Method): Array[Type] = m.getGenericParameterTypes
-  private[this] def parameterTypes(c: Constructor[_]): Array[Type] = c.getGenericParameterTypes
+  private[this] def parameterTypes(m: Method): Array[Type] =
+    ignoreNulls(m.getGenericParameterTypes)
+  private[this] def parameterTypes(c: Constructor[_]): Array[Type] =
+    ignoreNulls(c.getGenericParameterTypes)
```

Comment:
Did the previous iteration of this patch that only applied the filter on `getActualTypeArguments` fail the test in CI?

Reply:
Yes, the issue is that I forgot to push the latest local changes last night, which included this.